go.mod

```diff
@@ -3,11 +3,12 @@ module github.com/mattermost/mattermost-plugin-github
 go 1.12
 
 require (
-	github.com/Masterminds/sprig/v3 v3.0.2
-	github.com/google/go-github/v25 v25.1.1
+	github.com/Masterminds/sprig/v3 v3.1.0
+	github.com/google/go-github/v31 v31.0.0
 	github.com/gorilla/mux v1.7.4
-	github.com/mattermost/mattermost-server/v5 v5.18.0
-	github.com/pkg/errors v0.8.1
-	github.com/stretchr/testify v1.4.0
+	github.com/mattermost/mattermost-server/v5 v5.22.0
+	github.com/mholt/archiver/v3 v3.3.0
+	github.com/pkg/errors v0.9.1
+	github.com/stretchr/testify v1.5.1
```

Comment:
I did also include an dependency update into this PR to help keep the number of PRs QA has to review low.